pipeline/devel.groovy

```diff
@@ -7,7 +7,7 @@ pipelineStages = load 'infrastructure/pipeline/daily/stages.groovy'
 pipelineParameters = load 'infrastructure/pipeline/devel/parameters.groovy'
 
 
-def execute() {
+def execute(Boolean skipWithoutUpdates = false) {
```

Comment:
The name skipWithoutUpdates gives me the idea that the pipeline will skip without touching or updating anything. I would have called it skipIfNoUpdates.